hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -484,7 +505,7 @@ class SpecificMemberPms implements ProcessorMetaSupplier, IdentifiedDataSerializ
         SpecificMemberPms() {
         }
 
-        private SpecificMemberPms(ProcessorSupplier supplier, Address memberAddress) {
+        private SpecificMemberPms(ProcessorSupplier supplier, Address memberAddress, int localParallelism) {
```

Comment:
localParallelism is not used

Reply:
thanks, fixed https://github.com/hazelcast/hazelcast/pull/23831/commits/d1f93791d7890a65796e849c7853a9938d35b8c3